lib/auth/oauth2client.js

```diff
@@ -437,6 +437,10 @@ OAuth2Client.prototype.verifyIdToken = function(idToken, audience, callback) {
       'an ID Token and a callback method');
   }
 
+  if((typeof idToken) !=='string'){
```

Comment:
@StefanGussner Thanks for this addition! A quick nit:

* Since `lodash.isstring` is already included as a dependency to this lib -- to better conform with existing code would you mind changing this to look something like:

```JS
var isString = require('lodash.isstring');
// ...
if (!isString(idToken)) {
  throw new Error('...');
}
```